### ISSUE TITLE
PopoverController Bug

### ISSUE THREAD

**Issue by StormKid** [ionic-team/ionic#12895]
**Resources:**
Before submitting an issue, please consult our troubleshooting guide (http://ionicframework.com/docs/troubleshooting/) and developer resources (http://ionicframework.com/docs/developer-resources/)

**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[**x**] **3.x**

**I'm submitting a ...**  (check one with "x")
[**x**] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
I have used ' ionic cordova build android --prod ' even if I build my apk. I get the bug about this picture:
![image](https://user-images.githubusercontent.com/13798727/30465756-b084fcda-9a0b-11e7-85ff-7dc6952acd59.png)

**Expected behavior:**

**Steps to reproduce:**

**Related code:**

```
 ringClick(myEvent) {
    this.popoverCtrl.create(
      PopMsgComponent, {
        "tag": 0
      }
    ).present({
      ev: myEvent
    })
  }
```

**Other information:**
**_This Bug would like be the GPU bug. When I use MT6755 or Intel CPU it will be shown_**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```

```

**Comment by chandanch** [comment]
Can you share the template of _PopMsgComponent_?

**Comment by StormKid** [comment]
```
<ion-list inset>
    <button *ngIf="flag==0" ion-item (click)="sms()">私信</button>
    <button *ngIf="flag==0" ion-item (click)="sys()">系统消息</button>

    <button *ngIf="flag==1" ion-item (click)="update()">修改相册</button>
    <button style="color:red" *ngIf="flag==1" ion-item (click)="delete()">删除相册</button>

    <button style="color:red" *ngIf="flag==2" ion-item (click)="deletePhoto()">删除照片</button>

</ion-list>
```

**Comment by StormKid** [comment]
@chandanch

**Comment by StormKid** [comment]
_This bug only be showed on the phone that used MTK or Intel CPU_ @chandanch

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a small sample application via GitHub that demonstrates the issue you are having? Thanks for using Ionic!

**Comment by StormKid** [comment]
@kensodemann hello gentleman, look at this : 

# _**nubia z9mini**_
![nubia z9mini](https://user-images.githubusercontent.com/13798727/30527950-c1c4643e-9c60-11e7-8548-2da3411211c4.jpg)
```
操作系统Android 5.1
用户界面nubia UI 3.5
核心数真八核
CPU型号高通 骁龙615（MSM8939）
CPU频率1.5GHz（大四核），1.0GHz（小四核）
GPU型号高通 Adreno405
RAM容量3GB
ROM容量16GB
存储卡MicroSD卡
扩展容量128GB
电池类型不可拆卸式电池
电池容量2900mAh
理论通话时间2580分钟
理论待机时间750小时
```

# _**RedMI note4x**_
![note4x](https://user-images.githubusercontent.com/13798727/30528048-b79156b0-9c61-11e7-9d6a-7a00454fd595.png)
```
操作系统MIUI 8.1（基于Android 6.0）
用户界面MIUI 8.1
核心数真八核
CPU型号：高通 骁龙625（MSM8953）
CPU频率：2.0GHz
RAM容量4GB
ROM容量64GB
存储卡MicroSD卡
扩展容量128GB
电池类型不可拆卸式电池
电池容量4100mAh
理论通话时间3060分钟（2G）
理论待机时间264小时（4G）
```

# _**oppoR9m**_
![opporr9m](https://user-images.githubusercontent.com/13798727/30528086-1b0d9abe-9c62-11e7-80ae-00842c063cf4.jpg)
```
操作系统Color OS 3.0（基于Android 5.1）
核心数真八核
CPU型号联发科 ：MT6755
CPU频率：2.0GHz
GPU型号：MaliT-860 MP2
RAM容量4GB
ROM容量64GB
存储卡MicroSD卡
扩展容量128GB
电池类型不可拆卸式电池
电池容量2850mAh
其他硬件参数支持VOOC闪充，0.2秒极速解锁，1600w前置臻美自拍摄像头
```

# _**HuaWei Honour v9**_
![v9](https://user-images.githubusercontent.com/13798727/30528097-32817a6c-9c62-11e7-9aeb-7833f87e0b41.jpg)
```
操作系统EMUI 5.1（基于Android 7.0）
用户界面EMUI 5.1
核心数八核
CPU型号海思： Kirin 960
CPU频率2.4GHz（大四核），1.8GHz（小四核）+微智核I6
GPU型号：Mali G71 MP8
RAM容量4GB
ROM容量64GB
存储卡MicroSD卡
扩展容量128GB
电池类型不可拆卸式电池
电池容量4000mAh
理论通话时间1800分钟（3G）
理论待机时间504小时
其他硬件参数本地视频播放：约18小时
本地音乐播放：约108小时
支持9V/2A快速充电	
```
# _**VM**_
![default](https://user-images.githubusercontent.com/13798727/30528163-de22ca56-9c62-11e7-91a6-d9741e510b2a.png)
```
模拟器版本: 5.1.0.1
系统版本: Windows10
系统类型: 64
CPU: Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz
CPU核数: 4
总内存: 8100MB
可用内存: 1750MB
是否开启VT: 开启
渲染模式: OpenGL
OpenGL渲染: Intel(R) HD Graphics 4600
是否独显: 开启
显卡1: Intel(R) HD Graphics 4600
防火墙: 开启(推荐关闭)
```

**Comment by StormKid** [comment]
@kensodemann MY demo url: https://github.com/StormKid/popBugDemo and GIT url: https://github.com/StormKid/popBugDemo.git  .
Please use ' ionic cordova build android --prod' and starup apk on equipment that use MTK or Intel CPU.

**Comment by kensodemann** [comment]
Thank you for the sample application.

**Comment by wilfredonoyola** [comment]
Hi All,

I have same bug, How do I can fix it?

Thanks

**Comment by StormKid** [comment]
Please download ionic with lastest version. This bug had been resolved at ionic 3.18.1

**Comment by StormKid** [comment]
@wilfredonoyola Also you could use IOS mode
